WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -30,22 +33,25 @@
 #import "BlogDetailHeaderView.h"
 #import "ReachabilityUtils.h"
 
-const typedef enum {
+typedef NS_ENUM(NSInteger, BlogDetailsRow) {
     BlogDetailsRowViewSite = 0,
```

Comment:
`BlogDetailsRow` doesn't seem to ever be referenced as a type.  We could totally make those individual consts if that's better :)